app/src/main/java/org/wikipedia/analytics/metricsplatform/ApplicationData.kt

```diff
@@ -0,0 +1,15 @@
+package org.wikipedia.analytics.metricsplatform
+
+import org.wikipedia.WikipediaApp
+
+object ApplicationData {
+
+    val data = mapOf(
+        "app_theme" to WikipediaApp.instance.currentTheme,
+        "language_groups" to WikipediaApp.instance.languageState.appLanguageCodes
+    )
+
+    fun getApplicationData(): Map<String, Any> {
```

Comment:
In Kotlin it's no longer a convention to have a separate getter method that just returns a field. The `data` field itself can be referenced as `ApplicationData.data`.

Reply:
cool